image_tools/src/burger.cpp

```diff
@@ -124,7 +124,7 @@ cv::Mat & Burger::render_burger(size_t width, size_t height)
     }
     burger_buf = cv::Mat(height_i, width_i, CV_8UC3);
   }
-  burger_buf = cvScalar(0, 0, 0);
+  burger_buf = cv::Scalar_<int>(0, 0, 0);
```

Comment:
Is there any reason you're using `int` specifically rather than `double`, i.e. `cv::Scalar`? I assume there is, but I'm just curious.

Reply:
Oops, no. I think this should just be `cv::Scalar` :grimacing: 

Fixed in 2877adb